lms/envs/devstack.py

```diff
@@ -128,3 +130,6 @@
 #####################################################################
 # Lastly, run any migrations, if needed.
 MODULESTORE = convert_module_store_setting_if_needed(MODULESTORE)
+
+# Set this to an actual value to enforce API security
+EDX_API_KEY = None
```

Comment:
I think for development, it's nice to not enforce this for testing with curl/wget.